static/app/components/infoTooltip.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from '@emotion/styled';
+
+import type {TooltipProps} from 'sentry/components/tooltip';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconInfo} from 'sentry/icons';
+import type {SVGIconProps} from 'sentry/icons/svgIcon';
+import type {IconSize} from 'sentry/utils/theme';
+
+interface InfoTooltipProps
+  extends Partial<
+    Pick<
+      TooltipProps,
+      'containerDisplayMode' | 'isHoverable' | 'overlayStyle' | 'position'
+    >
+  > {
```

Comment:
instead of extending tooltip props, i'd prefer to see a prop specific to tooltipProps instead of several being available at the root.  

similar to this one https://github.com/getsentry/sentry/blob/master/static/app/components/button.tsx#L81-L84